### ISSUE TITLE
Can I set cb, cba[1], etc manually?

### ISSUE THREAD

**Issue by VirtualCartographer** [Strdate/PythonConsole#10]
Something like...

`cb = (0,0)`

**Comment by Strdate** [comment]
No, you can't, but I might add this feature. What would you use it for?

**Comment by VirtualCartographer** [comment]
When the cb is a whole number it makes debugging much simpler. So even rounding it into integer values could do the trick.

**Comment by VirtualCartographer** [comment]
Oof course Now that I think about it I can just round it myself
